presto-main/src/test/java/com/facebook/presto/operator/aggregation/AggregationTestUtils.java

```diff
@@ -238,22 +262,24 @@ public static Object partialAggregation(InternalAggregationFunction function, do
     public static Object partialAggregation(InternalAggregationFunction function, double confidence, int[] args, Page... pages)
     {
         AccumulatorFactory factory = function.bind(Ints.asList(args), Optional.empty(), Optional.empty(), confidence);
-        Accumulator partialAggregation = factory.createAccumulator();
+        Accumulator finalAggregation = factory.createIntermediateAccumulator();
```

Comment:
the declaration of this variable should be move to be immediately before it's use